### ISSUE TITLE
dao act: Cannot read property 'abi' of undefined

### ISSUE THREAD

**Issue by xuhcc** [aragon/aragon-cli#927]
## 🐛 Bug Report

`dao act` command started to fail after upgrading to version 6.4.0.

### Have you read the [Contributing Guidelines on issues](https://github.com/aragon/aragon-cli/blob/master/CONTRIBUTING.md#ways-to-contribute)?

Yes.

#### Mainnet or testnet?

Any.

#### Organization

Any organization with Agent. For example: https://rinkeby.aragon.org/#/0xA2d522f306cC8b42E9367a43fe7dA46Aef215d08/

#### Environment

- OS and OS version: Arch Linux
- Browser and browser version: N/A
- Aragon CLI: 6.4.0

## To Reproduce

Run `dao act` with any agent, other parameters do not matter:

```
dao act 0xbfE2F524c39D03BC512522E2CE0df8eF5A8EBFC9 0x0000000000000000000000000000000000000000 test --environment aragon:rinkeby --debug
```

## Expected behavior

Command should run without errors.

## Actual Behavior

Command execution fails with the following error:

```
❯ aragonCLI version: 6.4.0
❯ argv: ["/usr/bin/node","/usr/bin/aragon","dao","act","0xbfE2F524c39D03BC512522E2CE0df8eF5A8EBFC9","0x0000000000000000000000000000000000000000","test","--environment","aragon:rinkeby","--debug"]
❯ Could not find 'arapp.json'. Using the default configuration to connect to rinkeby.
call-args before parsing []
call-args after parsing []
✖ Cannot read property 'abi' of undefined
❯ TypeError: Cannot read property 'abi' of undefined
    at abi (/usr/lib/node_modules/@aragon/cli/src/commands/dao_cmds/act.js:41:29)
    at encodeCalldata (/usr/lib/node_modules/@aragon/cli/src/commands/dao_cmds/act.js:81:38)
    at tryCatch (/usr/lib/node_modules/@aragon/cli/node_modules/regenerator-runtime/runtime.js:45:40)
    at Generator.invoke [as _invoke] (/usr/lib/node_modules/@aragon/cli/node_modules/regenerator-runtime/runtime.js:271:22)
    at Generator.prototype.(anonymous function) [as next] (/usr/lib/node_modules/@aragon/cli/node_modules/regenerator-runtime/runtime.js:97:21)
    at tryCatch (/usr/lib/node_modules/@aragon/cli/node_modules/regenerator-runtime/runtime.js:45:40)
    at invoke (/usr/lib/node_modules/@aragon/cli/node_modules/regenerator-runtime/runtime.js:135:20)
    at /usr/lib/node_modules/@aragon/cli/node_modules/regenerator-runtime/runtime.js:145:13
    at process._tickCallback (internal/process/next_tick.js:68:7)
```

**Comment by 0xGabi** [comment]
Fixed with #920

**Comment by 0xGabi** [comment]
For the record the syntax of the `dao act` command is for example: 
```
dao act <agent-app> <voting-app> "vote(uint256,bool,bool)" 1 true true --env aragon:rinkeby
```
